### PR TITLE
Add descriptive titles for Extensible Prime Generator challenge

### PR DESCRIPTION
**Pull request by KartikSoneji**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

Adds descriptive titles instead of just `primeGenerator should be a function.`.

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/extensible-prime-generator.english.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ The function should have two parameters. The first will receive <code>n</code> o
 tests:
   - text: <code>primeGenerator</code> should be a function.
     testString: assert(typeof primeGenerator === 'function');
-  - text: <code>primeGenerator</code> should be a function.
+  - text: <code>primeGenerator(20, true)</code> return an array with the first 20 prime numbers.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/extensible-prime-generator.english.md:34]:
```suggestion
  - text: <code>primeGenerator(20, true)</code> should return an array with the first 20 prime numbers.
```
Lost a word here.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by nhcarrigan** [comment]
As an aside, would it be more ideal to have the test `text` indicate the actual array that should be returned?
The first array is a little long, but not terribly so.

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/extensible-prime-generator.english.md (hunk 1 above)
Comment (KartikSoneji) [reply curriculum/challenges/english/10-coding-interview-prep/rosetta-code/extensible-prime-generator.english.md]: Thanks for catching that!

**Comment by KartikSoneji** [comment]
@nhcarrigan This is what it looks like, let me know if I should make the changes.

image

**Comment by nhcarrigan** [comment]
I'm not opposed to that, and I think it makes it more clear. 

I would definitely be open to other opinions though. 🙂

**Review by naomi-lgbt — APPROVED** [review]
LGTM!

**Review by RandellDawson — APPROVED** [review]
Looks good.

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**